src/bin/dolos/data/export.rs

```diff
@@ -11,6 +11,10 @@ pub struct Args {
     /// the path to export to
     #[arg(short, long)]
     output: PathBuf,
+
+    // Whether to include ledger
+    #[arg(long, action)]
+    include_chain: bool,
```

Comment:
The description seems wrong, but regardless, I think it would be a good idea to have a flag to include each store, chain and / or ledger in the export.